### PR TITLE
Locate undeclared universe error from Print Universes Subgraph

### PR DESCRIPTION
**Pull request by SkySkimmer**

### CODE REVIEW

**Comment by Alizter** [comment]
As an aside, how breaking of a change would it be to make `~loc` a required argument in `CErrors` so that no location has to be explicitly passed with `Loc.none` or something. That way we could be sure we don't have missing locations in error messages via the API.

**Comment by Alizter** [comment]
In that case, what about introducing a `Loc.todo` which would default to `Loc.none` and be a placeholder for a future review. That way we could distinguish between a missing and a purposefully missing location.

File: vernac/vernacentries.ml [hunk 1]
```diff
@@ -421,13 +421,12 @@ let dump_universes_gen prl g s =
...
     Exninfo.iraise reraise
 
-(* [XXX] EGJA: loc unused here *)
 let universe_subgraph ?loc kept univ =
   let open Univ in
   let parse q =
     try Level.make (Nametab.locate_universe q)
     with Not_found ->
-      CErrors.user_err Pp.(str "Undeclared universe " ++ pr_qualid q ++ str".")
+      CErrors.user_err ?loc Pp.(str "Undeclared universe " ++ pr_qualid q ++ str".")
```
Comment (ppedrot) [inline vernac/vernacentries.ml:429]: This looks wrong, the location should be the one of the universe rather than the one of the global command.

Comment (SkySkimmer) [reply vernac/vernacentries.ml]: changed

**Review by ppedrot — APPROVED** [review]

**Comment by ppedrot** [comment]
@coqbot run full ci

**Comment by ppedrot** [comment]
@coqbot merge now

**Pull request merged**